pulpcore/tests/functional/api/test_openapi_schema.py

```diff
@@ -0,0 +1,68 @@
+"""Test related to the openapi schema Pulp generates."""
+import asyncio
+import json
+
+import aiohttp
+import pytest
+import jsonschema
+
+from drf_spectacular.validation import JSON_SCHEMA_SPEC_PATH
+from jsonschema import ValidationError
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="session")
```
We should only load this once (and use monkeypatch for the modifying fixture).

Reply:
They are parallel safe, I'm marking them as such! Also the plugin_template PR now integrates this new pytest run w/ the `parallel` markers too so this will be used correctly by marking them here.